cvmfs/mountpoint.cc

```diff
@@ -1233,6 +1237,27 @@ bool MountPoint::CreateDownloadManagers() {
   return SetupExternalDownloadMgr(false);
 }
 
+bool MountPoint::CreateResolvConfWatcher() {
+  if (!options_mgr_->IsDefined("CVMFS_DNS_ROAMING") ||
```

Comment:
Let's make the default off, no?  We can explicitly enable it then in mount/default.d/10-osx.conf